internal/api/v1/user/transactions/transactionstest/transactionstest.go

```diff
@@ -9,6 +9,146 @@ import (
 	"github.com/bitcoin-sv/spv-wallet/models/response"
 )
 
+func ExpectedDraftTransactionWithWrongInputs(t *testing.T) *response.DraftTransaction {
+	draftWithWrongInputs := ExpectedDraftTransactionWithHex(t)
+	draftWithWrongInputs.Configuration.Inputs[0].TransactionID = "wrong-input-transaction-id"
+	return draftWithWrongInputs
+}
+
+func ExpectedDraftTransactionWithWrongLockingScript(t *testing.T) *response.DraftTransaction {
+	draftWithWrongLockingScript := ExpectedDraftTransactionWithHex(t)
+	draftWithWrongLockingScript.Configuration.Inputs[0].Destination.LockingScript = "wrong-locking-script"
+	return draftWithWrongLockingScript
+}
+
+func ExpectedDraftTransactionWithWrongHex(t *testing.T) *response.DraftTransaction {
+	draftWithWrongHex := ExpectedDraftTransactionWithHex(t)
+	draftWithWrongHex.Hex = "wrong-hex"
+	return draftWithWrongHex
+}
+
+func ExpectedDraftTransactionWithHex(t *testing.T) *response.DraftTransaction {
+	return &response.DraftTransaction{
+		Model: response.Model{
+			CreatedAt: ParseTime(t, "2024-12-02T12:04:33.855018Z"),
+			UpdatedAt: ParseTime(t, "2024-12-02T13:04:33.855036+01:00"),
+			Metadata:  nil,
```

Comment:
Similar to other test stubs, there's no need to explicitly define nil values, as nil is the zero value for maps, channels, pointers, slices, etc. This can simplify the code and avoid redundancy.